doc/editor-integration.md

```diff
@@ -228,8 +228,8 @@ and CLJC (`.cljc`)<sup>1</sup>.
 4. Scope: `Current file`
 5. In the Program field, type `clj-kondo`
 6. In the Arguments field, type `--lint $FilePath$`<br>
-You may use a custom config E.g `--lint $FilePath$ --config "{:lint-as {manifold.deferred/let-flow clojure.core/let}}"`
-7. In the Working directory field, type `$FileDir$`
+You may use a custom config E.g `--lint $FilePath$ --config "{:lint-as {manifold.deferred/let-flow clojure.core/let}}"` or `--lint $FilePath$ --config your-custom-path/clj-kondo.conf.js"`
```

Comment:
The .js extension seems wrong. In hindsight, I think it's better to remove the config description, as this should be apparent from the general usage of clj-kondo.

Reply:
you are right about the extension, it's a typo